collections/luci-lib-docker/luasrc/docker.lua

```diff
@@ -307,7 +307,7 @@ local gen_api = function(_table, http_method, api_group, api_action)
 	local _api_action
 
 	if api_action == "get_archive" or api_action == "put_archive" then
-		api_action = "archive"
+		_api_action = "archive"
```

Comment:
I don't understand this line. The commit you used as a basis did have this line. https://github.com/lisaac/luci-lib-docker/commit/00e4888514e7b9190ae9be91d97892a4097f2fa4